### PR TITLE
Add include filter

### PR DESCRIPTION
**Pull request by daamien**

### CODE REVIEW

**Comment by daamien** [comment]
Closes #89

File: tests/test.sh [hunk 1]
```diff
@@ -94,6 +94,10 @@ $PANDOC --pdf-engine=xelatex $IN/magicienletter.md -o $OUT/magicienletter.html
 # 12. Dokuwiki
 $PANDOC --from dokuwiki --to markdown $IN/syntax.dokuwiki.txt -o $OUT/syntax.dokuwiki.md
 
+# 13. Include
+$PANDOC --filter pandoc-include $IN/include.md -o $OUT/include.complete.md
+
```
Comment (bersace) [inline tests/test.sh]:
``` bash
grep *included* $OUT/include.complete.md
```

:question:

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,5 @@
 panflute
 pandocfilters
-pandoc-include
```
Comment (bersace) [inline requirements.txt]: :question:

Comment (daamien) [reply requirements.txt]: my bad...

Comment (daamien) [reply requirements.txt]: fixed

**Pull request merged**